stabilizer/src/main/java/com/hazelcast/stabilizer/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,239 @@
+package com.hazelcast.stabilizer.utils;
+
+import com.hazelcast.stabilizer.test.annotations.Name;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.LinkedList;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class ReflectionUtils {
```

Comment:
This is mostly moved code from `TestContainer`, nothing really new here (some new wrapper functions with JavaDoc). Those methods are implicitly tested via `TestContainerTest`. But yes, I could add some more tests.